packages/dmn-js-shared/src/base/Manager.js

```diff
@@ -169,6 +169,11 @@ export default class Manager {
         return reject(parseError);
       }).then((result) => {
 
+        // break promise chain if we previously rejected or options.open was falsy
+        if (!result) {
```

Comment:
Do we need a test case for this?

Reply:
I discussed with Maciej -> we came to the conclusion to not do that but to simply avoid multipe chained `.then`s. This way we cannot get in the situation, that I tried to work around with this check.

Reply:
Commit got replaced by a simplification of the promise chain.

Reply:
BTW: intersting hour-of-code topic - I will add it for next week